app/views/admin/organisations/_form.html.erb

```diff
@@ -28,5 +28,26 @@
         %>
       </div>
     </div>
+    <h3><%= t('adapter_show_page') %></h3>
+    <div class="form-group">
+      <%= f.fields_for :adapter do |ff| %>
+        <%= ff.label(:countries_with_access_ids, class: 'col-sm-3 control-label'){
+            t('organisation_countries')
+          }
+        %>
+        <div class="col-sm-4">
+          <%= ff.select :countries_with_access_ids,
+                options_for_select(@countries_for_dropdown),
```

Comment:
shouldn't this be using `@available_countries` instead? `@countries_for_dropdown` is the unfiltered list of all countries.